front/lib/api/assistant/conversation.ts

```diff
@@ -584,6 +584,31 @@ export async function* postUserMessage(
   const subscription = auth.subscription();
   const plan = subscription?.plan;
 
+  const activeUsers = await countActiveSeatsInWorkspaceCached(
```

Comment:
Should come after the owner check below

Reply:
Actually let's move rate limits in API endpoint. Repeats a bit of code but really this is the endpoint responsibility not the internal lib 🙏